librz/arch/isa/msp430/msp430_disas.h

```diff
@@ -78,22 +99,48 @@ enum msp430_registers {
 	MSP430_R15,
 };
 
+typedef enum Msp430AddressingMode {
+	// register: Rn, contents of Rn
+	MSP430_REG,
+	// indexed: offset(Rn), contents of Memory[offset + Rn]
+	MSP430_INDX,
+	// symbolic: offset, contents of Memory[offset + PC] (as if indexed with Rn = PC)
+	MSP430_SYM,
+	// absolute: &addr, contents of Memory[addr] (as if indexed with a zeroed Rn)
+	MSP430_ABS,
+	// indirect register: @Rn, contents of Memory[Rn] (as if indexed with offset = 0)
+	MSP430_IND_REG,
+	// indirect register auto-increment: @Rn+, same as with indirect register but automatically increments Rn
+	MSP430_IND_AUTOINC,
+	// immediate: #literal, the literal value itself is the argument
+	MSP430_IMM
+} Msp430AddressingMode;
+
 struct msp430_cmd {
 	ut8 type;
 	ut8 opcode;
+
 	st16 jmp_addr;
-	ut16 call_addr;
 	ut8 jmp_cond;
 
 	// Null-delimited string representation of an assembly operation mnemonic.
 	// Length of array: 'i', 'n', 'v', 'a', 'l', 'i', 'd', '\0'
 	// (This is longer than any real assembly mnemonic.)
 	char instr[7 + 1];
+	// does it have a .b ?
+	bool is_byte;
 
 	// Null-delimited string representation of assembly operands.
 	// Length of array: 2 * ('0', 'x', 4-digit hexadecimal numeral, '(', 'r', 2-digit
 	// decimal numeral, ')'), ',', ' ', '\0'
 	char operands[2 * (2 + 4 + 2 + 3) + 2 + 1];
+
+	// The source and the dst of the operands, along with their modes
+	// This info is contained in the strings above, but parsing strings to obtain it is ugly so we replicate it here
+	ut32 src;
+	ut32 dst;
+	Msp430AddressingMode src_mode;
+	Msp430AddressingMode dst_mode;
```

Comment:
For the whole `msp430_cmd` struct:
- Please `typedef` it.
- Document each member separately with `///< ...`
- Add a doxygen to it with
  ```
  /**
   * \brief ...
   */
  ```